### PR TITLE
Eldritch Government

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added Eldritch Government:
  - No vassal contracts, taxes, and levies based on opinion.
  - Only rulers of the same religious family can inherit.
  - Dark Authority instead of Crown Authority:
    - Frenzy decays over time so to be efficient they should do their mechanics.
    - High Frenzy gives you skills per stress level and development growth.
  - Get Frenzy on executions.
  - Get Frenzy on mind breaks.
  - Mind breaks can't cause death.
  - Attracts Void Being Knights and Guests with high Skills.
- Nerfed demonic race traits.
- Gave attributes to Nerubian, N'raqi, C'Thrax, Qiraji, Mantid race traits.
- Demonic and Eldritch AI executes prisoners more often.
- Demonic AI tortures prisoners more often.

# Tasks:
- [x] Add unique government bonuses and mention in the government concept
- [x] Add backgrounds

# How to test:
Test gameplay features, tell if they look balanced.

In the Debug mode, right-click Eldritch government rulers to Give and Lose Dark Frenzy fast.

<details>
<summary>Click to expand</summary>

image

</details>

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/600/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

### CODE REVIEW

**Review by LordSilvermane — APPROVED** [review]
Loc looks good.

File: localization/english/wc_game_concepts_l_english.yml [hunk 1]
```diff
@@ -60,4 +60,7 @@
  game_concept_wc_incapable_desc:0 "Incapable characters cannot take actions independently due to severe illness or being isolated."
 
  game_concept_demonic_government:0 "$demonic_government_adjective$ Government"
- game_concept_demonic_government_desc:0 "A [dark_government|E] available to [GetTrait('being_demon').GetName( GetNullCharacter )] with $doctrine_disorder_magic_name$ accepted.\nHaving high [dark_authority|E], they get less [tyranny|E], slow down [dread|E] decay and their [armies|E] move faster."
\ No newline at end of file
+ game_concept_demonic_government_desc:0 "A [dark_government|E] available to [GetTrait('being_demon').GetName( GetNullCharacter )] with $doctrine_disorder_magic_name$ accepted.\nHaving high [dark_authority|E], they get less [tyranny|E], slow down [dread|E] decay and their [armies|E] move faster."
```
Comment (TheGroxEmpire) [inline localization/english/wc_game_concepts_l_english.yml:63]: I'm not sure if Demonic still gain less Tyranny for higher Dark Authority level.

**Review by TheGroxEmpire — COMMENTED** [review]
All of the features seem to be working as intended:
- You gain Dark Frenzy from having Mind Breaks and Executions
- Mind break can no longer kill your characters
- Void being guests seem to come more often with high skilled ruler

Regarding the balance, AI rarely executes prisoners for the sake of it, nor they would keep high stress levels through multiple Mind Breaks. This government is seemingly impossible to keep its Dark Authority for AI. The means for generating them are things that an AI doesn't usually do.
Dark Frenzy gain from Mind Break is too low. You gain 15 Dark Frenzy for each Mind Break, while you gain 20 from executing someone. It is actually quite hard, even as a player, to consistently keep stress level high to experience Mind Break. Once you gain Mind Break, another one will occur in 5 years time if you are still stressed. Playing as AQ, Dark Frenzy decays for -4 every year. If we rely on 5 years of Mind Break, we'd gain a net loss of -4 Dark Frenzy every 5 years.
My suggestion:
- Make AI strive for stress levels and Mind Breaks. If we can't do that, we should introduce other means of generating Dark Frenzy that is more plausible for AI
- Buff Dark Frenzy gain from Mind Breaks

File: localization/english/wc_game_concepts_l_english.yml (hunk 1 above)
Comment (arithon) [reply localization/english/wc_game_concepts_l_english.yml:63]: But that's not the topic of this PR, is it? This is about Old Gods.

Comment (TheGroxEmpire) [reply localization/english/wc_game_concepts_l_english.yml:63]: > But that's not the topic of this PR, is it? This is about Old Gods.

You are right, it was in the last dark government PR. I'll put ahead a separate issue for it if @zumbak04 wants it that way.

**Review by arithon — APPROVED** [review]
loc LGTM

File: localization/english/wc_game_concepts_l_english.yml (hunk 1 above)
Comment (zumbak04) [reply localization/english/wc_game_concepts_l_english.yml:63]: > You are right, it was in the last dark government PR. I'll put ahead a separate issue for it if @zumbak04 wants it that way.

Nah, it's fine.

Comment (zumbak04) [reply localization/english/wc_game_concepts_l_english.yml:63]: Though, nah, I'll fix it on another branch.

**Comment by zumbak04** [comment]
> Regarding the balance, AI rarely executes prisoners for the sake of it, nor they would keep high stress levels through multiple Mind Breaks. This government is seemingly impossible to keep its Dark Authority for AI. The means for generating them are things that an AI doesn't usually do.

Increased execution and torturing chance in fbf6007dccb11ae4525841b7abf82e294e6dceaa.
Mind breaks are a different story since stress gaining is scripted in every vanilla file so it's veeery hard to change it. May suggest lowering Dark Frenzy decay for AI.

> Dark Frenzy gain from Mind Break is too low.

Increased it in c8064bddd78169f85a07b883c40eb74e36e6c0dd.

File: localization/english/wc_game_concepts_l_english.yml (hunk 1 above)
Comment (zumbak04) [reply localization/english/wc_game_concepts_l_english.yml:63]: Fixed in a5474e3d6e311e1a943ebed51772f83c64aea593.

**Review by TheGroxEmpire — APPROVED** [review]
fbf6007 seems to be working. I planted several insignificant prisoners for Ahn'Qiraj and they surely executed most of them after some time.

The new 30 gain of Dark Frenzy from Mind Break sounds fair to me.

I did observer test play on two start dates to see how the AI would fare in managing their Dark Authority level:

- 583 - 598 : One ruler on the 2nd level. The rest are hovering on 3rd.

- 605-620: I saw many Eldritch government rulers actually reaching the 4th level. One in 3rd level and AQ is on 2nd level alongside one other ruler.

All in all, I think the AI has a pretty good chance in staying at 3rd and sometimes reaching 4th now. That solves my main gripe on this, well done.

**Review by Rhysheart — APPROVED** [review]
Played for about an hour in debug mode, tested playing as Ahn'Qiraj. Did not have any issues or bugs myself.

**Pull request merged**